README.md

```diff
@@ -1,21 +1,25 @@
-This is a small windows script to help extract Wings! amiga disks from the GOG release.
+This is a small Windows script to help extract Wings! amiga disks and the Kickstart 1.2 firmware from the GOG release. Also possible to extract from Linux, as explained below.
 
 # Requirements
 
-7z should be in your path, I got this on Windows by installing 7zip
+To have [7-Zip](https://www.7-zip.org/) installed.
 
 # Instructions
 
-1. Install Wings!
-2. find the Wings.exe, typically in "C:\GOG Games\Wings! Classic"
-3. drag and drop it over the extractwings.bat file
-4. you should now have two .adf files wings-1.adf and wings-2.adf these are the game disks
+1. Open the command prompt and run the command `set PATH=%PATH%;C:\Program Files\7-Zip\` (change `C:\Program Files\7-Zip\` to where you installed 7-Zip, if you didn't install it in the default folder)
```

Comment:
For me installing 7zip added 7z to the path in Windows. Did this not happen for you?

Reply:
At least not in the virtual machine I used to test. Maybe I should add a note about the path being possibly set already?